module/documents/activity/mixin.mjs

```diff
@@ -10,23 +11,42 @@ export default Base => class extends PseudoDocumentMixin(Base) {
    * Configuration information for Activities.
    *
    * @typedef {PseudoDocumentsMetadata} ActivityMetadata
-   * @property {string} type   Type name of this activity.
-   * @property {string} img    Default icon.
-   * @property {string} title  Default title.
+   * @property {string} type                              Type name of this activity.
+   * @property {string} img                               Default icon.
+   * @property {string} title                             Default title.
+   * @property {object} usage
+   * @property {string} usage.chatCard                    Template used to render the chat card.
+   * @property {typeof ActivityUsageDialog} usage.dialog  Default usage prompt.
    */
 
   /**
    * Configuration information for this PseudoDocument.
    * @type {PseudoDocumentsMetadata}
    */
   static metadata = Object.freeze({
-    name: "Activity"
+    name: "Activity",
+    usage: {
+      chatCard: "systems/dnd5e/templates/chat/activity-card.hbs",
+      dialog: ActivityUsageDialog
+    }
   });
 
   /* -------------------------------------------- */
   /*  Properties                                  */
   /* -------------------------------------------- */
 
+  /**
+   * Is scaling possible with this activity?
+   * @type {boolean}
+   */
+  get canScale() {
+    if ( !this.consumption.scaling.allowed ) return false;
+    if ( !this.isSpell ) return true;
```

Comment:
I should adjust the documentation here, but this is looking at whether the activity can scale outside of normal spell scaling.

For non-spell items, this is determined entirely by the "Allow Scaling" checkbox.

For spells, it might actually make sense of hide the "Consumption Scaling" section entirely, since scaling will be based on spell-specific logic.

Reply:
Got it, that makes sense